cluster-autoscaler/go.mod

```diff
@@ -17,26 +17,27 @@ require (
 	github.com/google/go-querystring v1.0.0
 	github.com/jmespath/go-jmespath v0.0.0-20180206201540-c2b33e8439af
 	github.com/json-iterator/go v1.1.9
-	github.com/mesosphere/konvoy/auto-provisioning/apis v0.0.0
+	github.com/mesosphere/konvoy/auto-provisioning/apis v0.0.0-00010101000000-000000000000
+	github.com/mesosphere/konvoy/auto-provisioning/apiutils v0.0.0-20210114164719-caf08683a4a3 // indirect
 	github.com/mesosphere/konvoy/clientapis v0.0.0
 	github.com/prometheus/client_golang v1.1.0
 	github.com/satori/go.uuid v1.2.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.5.1
-	golang.org/x/crypto v0.0.0-20200220183623-bac4c82f6975
+	golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9
 	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
 	google.golang.org/api v0.15.1
 	gopkg.in/gcfg.v1 v1.2.0
 	gopkg.in/yaml.v2 v2.3.0
-	k8s.io/api v0.18.10
-	k8s.io/apimachinery v0.18.10
+	k8s.io/api v0.19.6
```

Comment:
let's see if this doesn't cause issues to the other calls. That is my only concern.